Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.yml

```
@@ -0,0 +1,421 @@
+      description: Company overall score in numeric form (55-100).
+      type: Number
+    - contextPath: SecurityScorecard.Company.Score.grade
+      description: Company overall score in letter grade form (A-F).
+      type: String
+    - contextPath: SecurityScorecard.Company.Score.last30days_score_change
+      description: Company overall score numeric change (±) in the last month.
+      type: Number
+    - contextPath: SecurityScorecard.Company.Score.industry
+      description: ndustry category of the domain.
+      type: String
+    - contextPath: SecurityScorecard.Company.Score.size
+      description: Company size, e.g. 'size_more_than_10000'
+      type: String
+  - name: securityscorecard-company-factor-score-get
+    description: Retrieve company factor score.
+    arguments:
+    - name: domain
+      description: Company domain.
+      required: true
+      default: false
+    - name: severity_in
+      isArray: true
+      description: Issue severity filter. Optional values can be positive, info, low,
+        medium, high. Can be comma-separated list, e.g. 'medium,high,positive'
+      required: false
+    outputs:
+    - contextPath: SecurityScorecard.Company.Factor.name
+      description: Factor name.
+      type: String
+    - contextPath: SecurityScorecard.Company.Factor.score
+      description: Factor score in numeric form (55-100)
+      type: Number
+    - contextPath: SecurityScorecard.Company.Factor.grade
+      description: Factor score in letter grade form (A-F)
+      type: String
+    - contextPath: SecurityScorecard.Company.Factor.Issue.type
+      description: Type of issue found
+      type: String
+    - contextPath: SecurityScorecard.Company.Factor.Issue.count
+      description: How many times the issue was found
+      type: Number
+    - contextPath: SecurityScorecard.Company.Factor.Issue.severity
+      description: Severity of the issue
+      type: String
+    - contextPath: SecurityScorecard.Company.Factor.Issue.total_score_impact
+      description: Contribution of issue on overall score
+      type: Number
+    - contextPath: SecurityScorecard.Company.Factor.Issue.detail_url
+      description: URL to the details of the issue
+      type: String
+    - contextPath: SecurityScorecard.Company.Factor.total
+      description: Number of factors returned
+      type: Number
+  - name: securityscorecard-company-history-score-get
+    description: Retrieve company historical scores
+    arguments:
+    - name: domain
+      description: Company domain, e.g. `google.com`
+      required: true
+      default: false
+    - name: from
+      description: Initial date for historical data. Value should be in format `YYYY-MM-DD`
+      required: false
+      default: false
+    - name: to
+      description: Initial date for historical data. Value should be in format `YYYY-MM-DD`
+      default: false
+    - name: timing
+      description: Timing granularity. Acceptable values are `weekly` or `daily`
```

No need to mention possible values as they are given in the UI. Also, it creates duplicates in the readme file later on (as the automation adds this info to re.
**Go over all PREDEFINED arguments and make sure you are not mentioning possible values**
```suggestion
      description: Timing granularity.
```